server/bin/pbench-index.py

```diff
@@ -105,6 +105,17 @@ def main(options, name):
                 - Raises an exception caught be a new, outer-most, try/except
                   block that does not have a finally clause (as you don't want
                   any code execution in the finally block).
+        4. Re-evaluate the list of tar balls to index after indexing of the
+            current tar ball has finished
+            - SIGHUP
+            - Report the current state of indexing
+                - Current tar ball being processed
+                - How many remaining
+                - How many have been completed
+                - Errors encountered
```

Comment:
Reporting errors encountered might be useful, but you're not doing that.

Reply:
I will add that

Reply:
@dbutenhof which error should I report? like there is also a report generation at the end of the indexing that also reports no. of errors. So, should I just report the no. of errors the same ways or should I give details as well? also if there will be an error at the time of collection of tarballs inside sighup we can report that as well.

Reply:
ok sure